project_internal_access_from_portal/tests/test_portal_internal_access.py

```
@@ -0,0 +1,57 @@
+# -*- coding: utf-8 -*-
```

Ruff (0.8.2)

UTF-8 encoding declaration is unnecessary

Remove unnecessary coding comment

(UP009)